pages/checkout/_subscription.html

```diff
@@ -4,7 +4,7 @@
       <div class="col-md-8 panel panel-default panel-light checkout-panel-left">
         <div class="panel-body">
           <div class="row checkout-users">
-            <div class="col-sm-8 col-sm-offset-2">
+            <div class="col-sm-10 col-sm-offset-1">
```

Comment:
Why make this wider?

Reply:
"1 - 10" is wider than "1 - 5". Without the widening, this text was wrapping.

Reply:
Ah, gotcha